### ISSUE TITLE
How can I detect anomaly at pixel-level?

### ISSUE THREAD

**Issue by nnajeh** [tSchlegl/f-AnoGAN#16]

**Comment by nickk124** [comment]
The paper describes using pixel-wise residuals between the input/ground truth and the reconstruction.

**Comment by nnajeh** [comment]
@nickk124 Do you mean the same as the image-level residual MSE(fake_img, real_img) because the function not cleat at the paper?

**Comment by nickk124** [comment]
Yes, I think so. See item (4) in section 3.1 of the f-AnoGAN paper, as well as equation (5).

**Comment by nnajeh** [comment]
Thank you for your help.

**Comment by nnajeh** [comment]
@nickk124 I think the equation of,pixel_level anomaly detection is the loss L1 NOT MSE,  l1(fake_imgs, real_imgs) because in the paper A(x) =|x- G(E(x))|.
What do you think please, Thank you.
